src/ocean/meta/types/Arrays.d

```diff
@@ -107,3 +107,30 @@ unittest
     static assert (is(StripAllArrays!(int[float][4][][]) == int[float]));
     static assert (is(StripAllArrays!(double) == double));
 }
+
+/*******************************************************************************
+
+    Params:
+        T = any type
+
+    Returns:
+        If T is a static array, evaluates to matching dynamic array. Othwerwise
+        evaluates to just T.
+
+*******************************************************************************/
+
+public template SliceIfStaticArray ( T )
+{
+    static if (isArrayType!(T) == ArrayKind.Static)
+        alias ElementTypeOf!(T)[] SliceIfStaticArray;
+    else
+        alias T SliceIfStaticArray;
+}
+
+///
```

Comment:
This is a test, really, not a usage example. It'd be interesting to see an actual usage example (i.e. a situation where this template is useful), as its usefulness is not immediately obvious.